cmd/utils/flags.go

```diff
@@ -106,6 +106,12 @@ var (
 		Usage:  "Data directory for the databases and keystore. This value is only used in local DB.",
 		EnvVar: "KLAYTN_DATADIR",
 	}
+	ChainDataDirFlag = DirectoryFlag{
+		Name:   "chaindatadir",
+		Value:  DirectoryString{""},
+		Usage:  "Data directory for chaindata. If this is not specified, chaindata is stored in datadir",
```

Comment:
If this flag and rocksdb.secondary are set, does the node use only RocksDB?
A bit confused for me. If it is true, I think it's better to make a mode flag like KES.

Overall this PR, I can see only checking whether the RocksDB is used or not rather than checking something like a mode for the auto-scaling.

Reply:
@kjeom I think we can consider `ChainDataDir` flag to make data separately apart from other data (e.g. nodes, klay.ipc...). No need to consider rocksdb for this flag. And what do you mean by mode for the auto-scaling?

Reply:
Auto-scaling means what we named scalable endpoint.
Anyway now I understood the primary, read-only and secondary are options only for RocksDB.
I confused it as it is concerned with the primary and secondary for Klaytn.
Thanks